x-pack/plugins/security_solution/public/users/pages/types.ts

```diff
@@ -7,23 +7,24 @@
 import { ActionCreator } from 'typescript-fsa';
 
 import { GlobalTimeArgs } from '../../common/containers/use_global_time';
-import { UEBA_PATH } from '../../../common/constants';
-import { uebaModel } from '../../ueba/store';
+
+import { usersModel } from '../../users/store';
 import { DocValueFields } from '../../../../timelines/common';
 import { InputsModelId } from '../../common/store/inputs/constants';
+import { USERS_PATH } from '../../../common/constants';
 
-export const uebaDetailsPagePath = `${UEBA_PATH}/:detailName`;
+export const usersDetailsPagePath = `${USERS_PATH}/:detailName`;
```

Comment:
why this is `const` defined in a _types_ file?

Reply:
🤷